src/histolab/slide.py

```diff
@@ -120,47 +129,53 @@ def wsi_name(self) -> str:
 
     # ---private interface methods and properties---
 
-    def _breadcumb(self, directory_path) -> str:
+    def _breadcumb(self, directory_path, scale_factor=32) -> str:
         """Returns a complete path according to the give directory path
 
         Parameters
         ----------
         directory_path: str
+        scale_factor : int, default is 32
+            Image scaling factor
```

Comment:
I would align the param description under the type:
```
scale_factor : int, default is 32
               Image scaling factor
```

Reply:
Well, most of the examples of the Numpy style of docstrings simply start the description at one indentation level wrt the parameter name:
* https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard
* https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#module-sphinx.ext.napoleon
* https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html#example-numpy

Otherwise, you'll have the descriptions starting at different columns depending on how long are the parameters name. 

IMHO, having the description at one indentation level for all the parameters is a feature and not a bug :D

Reply:
as a matter of legibility, I do prefer the shape I suggest, but ok to me to use the one you prefer ,,, but we need to use that convention in all our module/packages

Reply:
As another example, docstring generated by `Python Docstring Generator` extension of VS Code: 

![image](https://user-images.githubusercontent.com/20052362/81623378-a3a42100-93f3-11ea-9325-fcf4590561f6.png)

I would prefer to be consistent with the convention out there

Reply:
Ok, I see, you wanna follow third party things instead of your mental pattern (or mine) ... 
OK TO ME  👯‍♀️